src/mesh/mesh.h

```diff
@@ -193,7 +194,7 @@ class Mesh {
     /**
      *
      */
-    void read_gmsh_from_stream(istream &in);
+    void read_gmsh_from_stream(istream &in, bool close_region_db = true);

```

Comment:
Why we need additional parameter? 
This method is used mainly in unit tests. Provide appropriate documentation.